build.rs

```diff
@@ -0,0 +1,65 @@
+use std::env;
+use std::path::PathBuf;
+
+extern crate cmake;
+use cmake::Config;
+
+fn main() {
+    let dst = Config::new(".")
+        .define("BUILD_STATIC_LIBS", "ON")
+        .build_target("MQF")
```

Comment:
@mr-eyes I ended up reverting the CMake changes, this was enough to avoid all the other targets (including binaries and tests)